fetch.bs

```diff
@@ -4518,6 +4521,14 @@ steps:
      <var>response</var>'s <a for=response>has-cross-origin-redirects</a> is true; otherwise
      <var>response</var>'s <a for=response>status</a>.
 
+     <li><p>Set <a for=response>body info</a>'s <a for="response body info">content type</a> to the
+     empty string if <var>response</var>'s <a for=response>type</a> is "<code>opaque</code>" or
```

Comment:
As with the other check we probably don't need to check "opaque" here as "opaque" responses won't have a `Content-Type` header anyway? (Unless you poke at its internals, which we're not doing here if I remember correctly.)

Reply:
Oh right..Thanks. I've dropped the check for `opaque`.